contrib/babelfishpg_tsql/src/properties.c

```diff
@@ -203,6 +204,44 @@ get_product_version_helper(int idx)
     return (VarChar *)info;
 }
 
+static VarChar *
+get_product_level_helper()
+{
+	StringInfoData	temp;
+	void		*info;
+	const char	*product_version;
+	int				minor_version;
+	char*			product_level_RTM = "RTM";
+	char*			product_level_prefix = "SP";
+	
+    initStringInfo(&temp);
+	product_version = GetConfigOption("babelfishpg_tds.product_version", true, false);
```

Comment:
I think we are going to use `BABELFISH_VERSION_STR`. Please double check with Rob.